corfudb-tools/src/main/java/org/corfudb/browser/CorfuStoreBrowser.java

```diff
@@ -0,0 +1,80 @@
+package org.corfudb.browser;
+
+import com.google.common.reflect.TypeToken;
+
+import java.util.Map;
+
+import lombok.extern.slf4j.Slf4j;
+import org.corfudb.runtime.CorfuRuntime;
+import org.corfudb.runtime.collections.CorfuTable;
+import org.corfudb.runtime.collections.CorfuDynamicKey;
+import org.corfudb.runtime.collections.CorfuDynamicRecord;
+import org.corfudb.runtime.view.TableRegistry;
+import org.corfudb.util.serializer.DynamicProtobufSerializer;
+import org.corfudb.util.serializer.ISerializer;
+import org.corfudb.util.serializer.Serializers;
+
+/**
+ * This is the CorfuTable Browser Tool which prints data in a given namespace and table.
+ *
+ * - Created by pmajmudar on 10/16/2019.
+ */
+@Slf4j
+public class CorfuStoreBrowser {
```

Comment:
The comment says CorfuTableBrowser and class name is CorfuStoreBrowser.
Fix either of them.

Reply:
changed comment to 'CorfuStoreBrowser'